.github/pull_request_template.md

```diff
@@ -0,0 +1,20 @@
+# Related Issue
+- issue goes here
+
+# Proposed Changes
```

Comment:
```suggestion
** Details **
```

use bold instead of h1 headings, these would be less intrusive

Reply:
I had committed your suggestion

Reply:
Please do it across the whole file, there are other headings as well :)

Reply:
> Please do it across the whole file, there are other headings as well :)

@ssgshubham

Reply:
Okay, on it, just saw it now.

Reply:
Please check, whether it is right or not, now